modules/developer-lightspeed/proc-installing-and-configuring-lightspeed.adoc

```diff
@@ -526,7 +527,43 @@ spec:
       configMaps:
         - name: rbac-policies
 ----
-For detailed information, see {authorization-book-link}managing-authorizations-by-using-external-files[Managing authorizations by using external files].
+For detailed information, see {authorization-book-link}##con-rbac-conditional-policies-rhdh_title-authorization[Managing authorizations by using external files].
+
+.. For a Helm-installed {product-very-short} instance,  define permissions and roles for your users who are not administrators by completing the following steps:
+... The RBAC plugin is installed but disabled by default. To enable the `./dynamic-plugins/dist/backstage-community-plugin-rbac plugin`, edit your `dynamic-plugins.yaml` with the following content:
++
+[source,yaml]
+----
+plugins:
+  - package: ./dynamic-plugins/dist/backstage-community-plugin-rbac
+    disabled: false
```

Comment:
This is to manage RBAC policy via RBAC UI, since we are using CSV file approach here, you can remove this enabling UI plugin instruction.